src/makeCalculator.js

```diff
@@ -37,7 +37,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let n = 0;
+
+  const calculator = {
+    operate(callback, y) {
+      callback.call(this, this.result, y);
+
+      return this;
+    },
+    add(x, y) {
```

Comment:
you can pass only one param to your methods and make it work like this:

```javascript
this.result += x
``` 

fix everywhere